File: Prototype Pattern/1. Minimal/Java/ConcretePrototype2.java
public class ConcretePrototype2 implements Prototype {
    private String value;

    public ConcretePrototype2(String value) {
        this.value = value;
    }

    @Override
    public Prototype clone() {
        return new ConcretePrototype2(this.value);
    }

    @Override
    public String toString() {
        return "value = '" + this.value + "'";
    }
}

File: Prototype Pattern/1. Minimal/Python/concrete_prototype_2.py
from prototype import Prototype

class ConcretePrototype2(Prototype):
    def __init__(self, value):
        self._value = value

    def clone(self):
        return ConcretePrototype2(self._value)
    
    def __str__(self):
        return "value = '" + self._value + "'"
